packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -38,7 +37,6 @@ describe('Tooltip', () => {
   it('uses default documented properties', () => {
     const component = mount(<TooltipBase />);
 
-    expect(component.prop('delay')).toEqual(TooltipDelay.medium);
```

Comment:
Now that `Tooltip` is not simulating a delayed render through `animationDelay` we are not defaulting this prop